oci/client/build.go

```diff
@@ -50,8 +56,8 @@ func (c *Client) Build(artifactPath, sourceDir string, ignorePaths []string) (er
 
 	ignore := strings.Join(ignorePaths, "\n")
 	var domain []string
-	if sourceDir != "." {
-		domain = strings.Split(filepath.Clean(sourceDir), string(filepath.Separator))
+	if absDir != "." {
```

Comment:
> My understanding is that absolute path will always at least have `/`, root.

It could start with `C:\Users` on Windows. As for using `os.Getwd()` what if the path is to dir on `D` drive but the working drive is `C`?

Reply:
I have removed the check. The check was introduced here https://github.com/fluxcd/pkg/pull/314 and it's no longer needed since we get the absolute file paths now and we can use that for as the domain